### PR TITLE
fix for DNA raw reads counts including RNA

### PR DESCRIPTION
**Pull request by Buwujiu**

### CODE REVIEW

File: src/elasticsearch/donorIndexMapping.json [hunk 1]
```diff
@@ -51,6 +51,12 @@
       "registeredTumourSamples": {
         "type": "byte"
       },
+      "RNApublishedNormalAnalysis": {
```
Comment (Buwujiu) [inline src/elasticsearch/donorIndexMapping.json:54]: Adding two new fields for RNA raw reads columns(new feature for next sprint), not really in the scope of this ticket, but was a quick implementation and kinda related to this bug.

Comment (joneubank) [reply src/elasticsearch/donorIndexMapping.json:54]: Camelcase naming is weird here:
`rnaPublishedNormalAnalysis` and `rnaPublishedTumourAnalysis` looks better to my eyes

Comment (joneubank) [reply src/elasticsearch/donorIndexMapping.json:54]: Would update this throughout.

File: src/rdpc/analysesProcessor.ts [hunk 2]
```diff
@@ -231,10 +234,16 @@ export const getAllMergedDonor = async ({
...
+    const normalSpecimen = donorData.specimen.filter(
+      (sp) => sp.tumourNormalDesignation === TumourNormalDesignationValue.Normal
+    );
+
+    const tumourSpecimen = donorData.specimen.filter(
+      (sp) => sp.tumourNormalDesignation === TumourNormalDesignationValue.Tumour
+    );
+
+    for (const normal of normalSpecimen) {
+      if (normal.samples[0].sampleType.includes(DNA_SAMPLE_TYPE_KEYWORD)) {
```
Comment (joneubank) [inline src/rdpc/analysesProcessor.ts:246]: Is there any risk that `normal.samples[0]` will be null? Should this be `normal.samples[0]?.sampleType.includes(DNA_SAMPLE_TYPE_KEYWORD)`.

**Review by joneubank — CHANGES_REQUESTED** [review]
Good stuff. Found two things worth considering updating.

**Review by joneubank — APPROVED** [review]

**Pull request merged**